README.md

```diff
@@ -26,7 +26,41 @@ To run the full unit test suite, run:
 This will run the same tests that we run in the Travis CI build.
 
 ### Deploying
-To deploy a personal stack:
+First, take a look at what Blox will put in your personal stack by running the
+`showStackConfig` task:
+
+```
+$ ./gradlew showStackConfig
+
+> Task :showStackConfig
+Blox deployment stack configuration:
+
+  Default resource name         (blox.name): blox-buys-alpha-us-west-2 (default)
```

Comment:
It's the default for all resource names. We can change the default and the stack name independently. Not sure if it's necessary, but considering we're not tightly coupling any resource names (very particularly AWS Profile) anymore, I figured keeping this decoupled too wouldn't hurt.